src/app/zap-templates/helper-chip.js

```diff
@@ -21,6 +21,25 @@
  * @module Templating API: toplevel utility helpers
  */
 
+// Import Zcl helper from zap core
+const helperZcl = require('../../../third_party/zap/repo/src-electron/generator/helper-zcl.js')
```

Comment:
Other projects may have a different source tree layout. Can we pass the location of third_party/zap/repo as an argument to the template?